backend/provisioner/testdata/go/echo/echo.go

```diff
@@ -8,7 +8,11 @@ import (
 	"github.com/TBD54566975/ftl/go-runtime/ftl"
 )
 
-var db = ftl.PostgresDatabase("echodb")
+type EchoDBConfig struct {
+	ftl.DefaultPostgresDatabaseConfig
+}
+
+func (EchoDBConfig) Name() string { return "echodb" }
```

Comment:
I think this is pretty good. It would get super messy if we tried to infer it from the type name, particularly with it being FooBar_Config_.